libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -610,6 +610,16 @@ AP_GPS::first_unconfigured_gps(void) const
 }
 
 void
+AP_GPS::broadcast_first_configuration_failure_reason(void) const {
+    uint8_t unconfigured = first_unconfigured_gps();
+    if(drivers[unconfigured] == NULL) {
```

Comment:
nitpick, but this doesn't conform to coding style... it's missing a space after `if`